Minutes for 12th Nov SSWG meeting (#16)
IATI/standard-process: meetings/2024-11-12.md

Apologies: Annelise Parr (UNDP), Ryan Boles (USAID),  Marcel Laqueur (FFO Germany), Johan Zambrano (USAID), Melina Lower (GIZ)

Non attendees: Wakilongo Mweshwa Robert (AIGD AFRICA)

### Minutes 

#### Recap of objectives and previous minutes

* [Recap](https://docs.google.com/presentation/d/1N6Xu8GZYte61yE5Gcpaty3rrrZXf_ho1mWmLOjIMPpI/edit?usp=sharing) of objectives of the SSWG from Steven.
* Previous minutes have been shared in [GitHub](https://github.com/IATI/standard-process/tree/main/meetings).
* Today is the final meeting of this time bound Standard Stewardship Working Group (SSWG).

#### Reflections on work so far

* Concern about short time to next board meeting and MA, and risk of not changing the standard for another year.
* Suggestion of small initial changes to the standard, to reintroduce the community to the concept of a changing data standard. 
* We have principles to work to from this group. 

#### Examples from other standards

Steven shared two examples of change control documents from other data standards,  [360 Giving](https://github.com/IATI/standard-process/discussions/10) and [Open Referral](https://github.com/IATI/standard-process/discussions/13).

> * Agreement that a similar template would be useful for IATI.

Reviewer: ```suggestion
* Agreement that similar templates for proposing changes would be useful for IATI.
```

* Need to ensure that this process includes communicating changes and their impact to data users.
* Idea - a requirement to "subscribe" to a IATI Standard Updates listserv in order to download IATI files or embed/use secretariat run APIs?
* A standard way of logging dependencies would be useful - i.e. which tools would be impacted by a change and how
* Suggestion of a standard update gazette.

#### Impact assessment

> Amy presented a draft [impact-based upgrade approach](https://github.com/IATI/standard-process/discussions/12). This approach would be pragmatic, and decouple changes to the standard from the Major/Minor/Patch definitions.

Reviewer: ```suggestion
Amy presented a draft [impact-based upgrade approach](https://github.com/IATI/standard-process/discussions/12). This approach would be pragmatic, based on real-world impact rather than theoretical or hypothetical, and decouple the decision-making about changes to the standard from the Major/Minor/Patch definitions.
```

* Is the Secretariat responsible for triaging issues and setting a timeline, or should the Secretariat coordinate a working group open to the community/members to do this?
* What is the bar for proposing changes to the Board to sign off? (eg. demonstrated implementations, or something else?)

Reflections:

* The work of this group means we are in a good position process wise.
* There is work to be done on the governance side of standard changes.
* Would look to make changes that aren't substantive next year - they could be "major" but wouldn't have a major impact.
* Useful to batch changes together using this approach.

#### Github Discussion post: Putting users at the centre of the change process 

[Github discussion](https://github.com/IATI/standard-process/discussions/15) shared by David. 

Noting that it would be useful to have defined guidelines around consulting users about changes, and sharing the results of these consultations.

#### Standard Operating Procedures (SOPs)

Recap of the relevant parts of the SOPs from Rob. We aren't in a place to propose changes to the SOPs yet.

Question: Can we ask to suspend the SOPs in this area to give flexibility to try new processes?

> * Agreement that SOPs can be changed by members, but existing SOPs should be followed for now.

Reviewer: ```suggestion
* Agreement that SOPs can be changed by members, but existing SOPs should be followed for now.
* We can work within the bounds of the SOPs, while experimenting with new processes for describing and evaluating the impact of changes.
```

* Secretariat should approach MA to communicate this work and plans, and to ask for goodwill to take this work forward. Board will then have oversight of the ongoing work.

> * Would be useful to set forward a 6 month timetable to inform the community.

Reviewer: ```suggestion
* Would be useful to set forward a 6 month timetable of anticipated changes to inform the community and get wider engagement in the process.
```
